samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Heading:<strong> Step 6: Transfer ...</strong></p><p> </p><p>The text does not explain what draft resources are. Perhaps we can rephrase the paragraph as below:</p><p>"</p><p><code>ArcGIS StoryMaps</code> support a workflow of editing a published story, previewing those changes, and then republishing for the changes to take effect. Such updates to published stories are stored as unpublished drafts and are not visible to the audience until you are ready to republish the story.</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Agreed. Changed the text.</p>